.goreleaser.yml

```diff
@@ -113,6 +113,9 @@ signs:
   - artifacts: checksum
     args: ["--batch", "-u", "{{ .Env.GPG_FINGERPRINT }}", "--output", "${signature}", "--detach-sign", "${artifact}"]
 
+sboms:
+  - artifacts: archive
+
```

Comment:
can we just make the goreleaser the single point to generate sboms so we can reproduce it locally with no other command execution?

Reply:
SBOM in goreleaser can only work with `goreleaser release` command. So in ci workflow, if we want to generate SBOM report, we can not use goreleaser (as we are using goreleaser build).